lib/beaker/tasks/quick_start.rb

```diff
@@ -1,49 +1,93 @@
 require 'beaker-hostgenerator'
 
+VAGRANT  = ['ubuntu1404-64default.mdcal-ubuntu1404-64af', '--hypervisor=vagrant',
+            '--global-config=box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm']
```

Comment:
> beaker-hostgenerator ubuntu1404-64default.mdcal-ubuntu1404-64af --hypervisor=vagrant --global-config={box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm}

You need to escape the `,`'s when invoking on the CLI, which is why it didn't produce the right thing.

FYI the behavior differs between the CLI and CI due to `{` and `}` being special characters the shell interprets rather than passing to BHG.  You can get around this by escaping either the surrounding braces or each comma within the braces.

> beaker-hostgenerator ubuntu1404-64default.mdcal-ubuntu1404-64af --hypervisor=vagrant --global-config=box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm

By golly you're right!  This is just a coincidence due to how it's written currently and not an API guarantee.

Yeah we should include the braces though since that's the API.